payout/params.go

```diff
@@ -4,6 +4,7 @@ package payout
 type CreateParams struct {
 	ExternalID string  `json:"external_id" validate:"required"`
 	Amount     float64 `json:"amount" validate:"required"`
+	Email      string  `json:"email" validate:"required"`
```

Comment:
wdyt?
```suggestion
	Email      string  `json:"email" validate:"required,email"`
```

Reply:
I don't think that's necessary, intentionally left out all the specific type check because I want our REST clients to handle checking for required fields only so that the error formation won't get too complicated

Reply:
ok 👍